playbook_examples/add_resource_in_resource_group.yml

```diff
@@ -45,8 +45,8 @@
     - name: Create a resource in squest
       uri:
         url: "{{ squest_api }}resource_tracker/resource_group/{{ resource_group_vm_id }}/resources/"
-        user: "admin"
-        password: "admin"
```

Comment:
if logn password is now disabled, this need to be done in each playbook and in the doc